CHANGELOG.md

```diff
@@ -1 +1,99 @@
++ Add "pid" to unsupported options list [docker/cli#768](https://github.com/docker/cli/pull/768)
++ Add support for experimental Cli configuration [docker/cli#758](https://github.com/docker/cli/pull/758)
++ Add support for generic resources to bash completion [docker/cli#749](https://github.com/docker/cli/pull/749)
+- Fix error in zsh completion script for docker exec [docker/cli#751](https://github.com/docker/cli/pull/751)
++ Add a debug message when client closes websocket attach connection [moby/moby#35720](https://github.com/moby/moby/pull/35720)
+
+### Deprecation
+
+* Updated deprecation status for "repository:shortid" [docker/cli#753](https://github.com/docker/cli/pull/753)
+* Remove support for referencing images by 'repository:shortid' [moby/moby#35790](https://github.com/moby/moby/pull/35790)
+
+### Documentation
+* Correct references to --publish long syntax in docs [docker/cli#746](https://github.com/docker/cli/pull/746)
+* Corrected descriptions for MAC_ADMIN and MAC_OVERRIDE [docker/cli#761](https://github.com/docker/cli/pull/761)
+* Updated developer doc to explain external CLI [moby/moby#35681](https://github.com/moby/moby/pull/35681)
+- Fix "on-failure" restart policy being documented as "failure" [docker/cli#754](https://github.com/docker/cli/pull/754)
+- Fix anchors to "Storage driver options" [docker/cli#748](https://github.com/docker/cli/pull/748)
+
+### Logging
+
+* Modify log in order to be same below [moby/moby#35567](https://github.com/moby/moby/pull/35567)
+* Update Graylog2/go-gelf vendoring. Fixes #35613 [moby/moby#35765](https://github.com/moby/moby/pull/35765)
+- Fix awslogs batch size calculation [moby/moby#35726](https://github.com/moby/moby/pull/35726)
+- Fix typo in log-message [moby/moby#35768](https://github.com/moby/moby/pull/35768)
+
+### Networking
+
+* [AUTOMATED] libnetwork Vendoring [moby/moby#35791](https://github.com/moby/moby/pull/35791)
+- Fix external networks in stacks [docker/cli#743](https://github.com/docker/cli/pull/743)
+
+### Runtime
+
+* Carry #35609 (remove import of opencontainers/runc in windows) [moby/moby#35788](https://github.com/moby/moby/pull/35788)
+* Devmapper: Log fstype and mount options during mount error [moby/moby#35732](https://github.com/moby/moby/pull/35732)
+* Windows: Case-insensitive filename matching against builder cache [moby/moby#35793](https://github.com/moby/moby/pull/35793)
+- Fix race conditions in libcontainerd process handling [moby/moby#35809](https://github.com/moby/moby/pull/35809)
+* Use commit-sha instead of tag for containerd [moby/moby#35770](https://github.com/moby/moby/pull/35770)
+* Disallow using legacy (V1) registries [moby/moby#35751](https://github.com/moby/moby/pull/35751)
+* Don't append the container id to custom directory checkpoints. [moby/moby#35694](https://github.com/moby/moby/pull/35694)
+* Re-validate Mounts on container start [moby/moby#35833](https://github.com/moby/moby/pull/35833)
```

Comment:
Can you use the description from "Description for changelog"?

```Markdown
* Validate Mount-specs on container start to prevent creating missing host-paths [moby/moby#35833](https://github.com/moby/moby/pull/35833)
```